test/unit/specs/events/cd-event-list.spec.js

```diff
@@ -241,6 +242,83 @@ describe('Event list component', () => {
         });
       });
     });
+    describe('isPastEvent', () => {
+      it('should return true for a one-off event whose date is before now', () => {
+        // ARRANGE
+        const vm = vueUnitHelper(EventListWithMocks);
+        const mockEvent = {
+          dates: [
+            {
+              startTime: '2017-06-10T16:30:00.000Z',
+            },
+          ],
+        };
+
+        // ACT & ASSERT
+        expect(vm.isPastEvent(mockEvent)).to.equal(true);
+      });
+      it('should return true for a recurring event whose last date is before now', () => {
+        // ARRANGE
+        const vm = vueUnitHelper(EventListWithMocks);
+        const mockEvent = {
+          type: 'recurring',
+          dates: [
+            {
+              startTime: '2017-06-10T16:30:00.000Z',
+            },
+            {
+              startTime: '2017-06-17T16:30:00.000Z',
+            },
+            {
+              startTime: '2017-06-24T16:30:00.000Z',
+            },
+          ],
+        };
+
+        // ACT & ASSERT
+        expect(vm.isPastEvent(mockEvent)).to.equal(true);
+      });
+      it('should return false for a one-off event whose date is after now', () => {
+        // ARRANGE
+        const vm = vueUnitHelper(EventListWithMocks);
+        const now = moment();
+        const startTime = now.add(1, 'day');
+        const mockEvent = {
+          dates: [
+            {
+              startTime,
+            },
+          ],
+        };
+
+        // ACT & ASSERT
+        expect(vm.isPastEvent(mockEvent)).to.equal(false);
+      });
+      it('should return false for a recurring event whose last date is after now', () => {
+        // ARRANGE
+        const vm = vueUnitHelper(EventListWithMocks);
+        const now = moment();
+        const startTime = now.add(1, 'day');
+
+        const mockEvent = {
+          type: 'recurring',
+          dates: [
+            {
+              startTime: '2017-06-17T16:30:00.000Z',
+            },
+            {
+              startTime: '2017-06-24T16:30:00.000Z',
+            },
+            {
+              startTime,
+            },
+          ],
+        };
+
+        // ACT & ASSERT
+        expect(vm.isPastEvent(mockEvent)).to.equal(false);
+      });
+    });
```

Comment:
We should have a test that tests timezones... I'll get back to you on that one 😅